### PR TITLE
added navBar Component.

### PR DESCRIPTION
**Pull request by HasancanKazan**

### CODE REVIEW

File: src/components/navbar/_navbar.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+.nav {
+    background-color: #333;
+    overflow: hidden;
+  }
+  
+  .nav a {
+    float: left;
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
```
Comment (bilginyuksel) [inline src/components/navbar/_navbar.scss]: use rem instead of px. Please check my pull request.

File: src/components/navbar/Navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                })}
+
+                {/* <a className="active" href="#">Survey List</a>
+                <a href="#">Create Survey</a>
+                <a href="#">My Surveys</a>
+                <a href="#">Profile</a>  */}
+
+            </div>
+            <br />
+        </div>
```
Comment (bilginyuksel) [inline src/components/navbar/Navbar.tsx]: You are using two `div` elements. Second `div` element is unnecessary you can use the parent `div`. And you should write a small function for a navigation element. And then just call the function while you iterating the array.  
```suggestion
      <div className="nav">
          <a href="#" className="logo" >Survey Genarator</a>
          {navbarItems.map((item)=>{
             return navigationElement(item);
          })}
    </div>
        
const navigationElement = (item) => {
    if(item.isActive)
        return <a className="active" href={item.url}>{item.description}</a>
    else
        return <a href={item.url}>{item.description}</a>
}
```

**Review by bilginyuksel — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please consider what I recommended to you and commit to the current pull request.

File: .vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
```
Comment (yasincaliskan) [inline .vscode/launch.json:5]: I'm not sure this file is necessary.

File: src/app/TimeLine.tsx [hunk 4]
```diff
@@ -12,6 +13,8 @@ export function TimeLine() {
 
   return (
     <div>
+      <Navbar/>
```
Comment (yasincaliskan) [inline src/app/TimeLine.tsx]: I think Navbar component should in the Home component. We can change after current PR's when merged.

**Review by bilginyuksel — COMMENTED** [review]
I resolved the conflicts.

**Review by yasincaliskan — CHANGES_REQUESTED** [review]

File: .vscode/launch.json (hunk 3 above)
Comment (bilginyuksel) [reply .vscode/launch.json:5]: Yes, I think we can add it to .gitignore.

File: src/app/TimeLine.tsx (hunk 4 above)
Comment (HasancanKazan) [reply src/app/TimeLine.tsx]: It was written here to serve as an example

File: .vscode/launch.json (hunk 3 above)
Comment (HasancanKazan) [reply .vscode/launch.json:5]: I added the file to .gitignore but now I can't see it

**Review by bilginyuksel — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**